calico/reference/installation/_crd-ref-docs/config.yaml

```diff
@@ -0,0 +1,69 @@
+processor:
```

Comment:
So it looks like you generate the API docs based on the Installation CRDs, presumably for a specific version of the Operator, which serves both OSS and CE. Here, you exclude CE-only items from the list. 

Have I got that right?

Reply:
Is there a way to automate this? I'm concerned about maintenance in the long term.